src/features/scheduling/BundleScheduler.ts

```
@@ -0,0 +1,57 @@
+  bundleNum: number = -1;
+  schedule: SectionSchedule<'BUNDLE'> = { blocks: {}, alternatePeriodsOff: {} };
+
+  campers: CamperAttendeeID[] = [];
+  staff: StaffAttendeeID[] = [];
+  admins: AdminAttendeeID[] = [];
+
+  camperPrefs: Preferences = {};
+
+  blocksToAssign: string[] = [];
+
+  constructor() { return this; }
```

Biome (2.1.2)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)